test/integration/services/data/test-get-contracted-hours-for-workload-owner.js

```diff
@@ -0,0 +1,22 @@
+const expect = require('chai').expect
+
+const dataHelper = require('../../../helpers/data/aggregated-data-helper')
+const getContractedHours = require('../../../../app/services/data/get-contracted-hours-for-workload-owner')
+
+var workloadOwnerId
+
+describe('services/data/get-contracted-hours-for-workload-owner', function () {
+  before(function () {
+    return dataHelper.getAnyExistingWorkloadOwnerId()
+      .then(function (id) {
+        workloadOwnerId = id
+      })
+  })
+
+  it('should retrieve contracted hours for a valid workload owner id', function () {
```

Comment:
Any failover for when a workload owner does not exist?  Or should this be handled any way by the catchall errors code

Reply:
Should be handled by the catchall.